Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -263,6 +269,8 @@
 
                             parameterMap.$filter =
                                 replaceSystemFilter(parameterMap.$filter, "AssociatedSystemUsages");
+                            parameterMap.$filter =
+                                replaceSystemUuidFilter(parameterMap.$filter, "ExhibitedBy.ItSystem.Uuid");
```

Comment:
Exhibited by is a property on ItInterface, so this will not work

Reply:
it works because the datasource prop is wrong but that on the other hand will make the excel export fail